### PR TITLE
Fix basics

### PR DESCRIPTION
**Pull request by MichaelBroughton**
Cleans up basics.ipynb and removes mention of old devices behavior.

### CODE REVIEW

File: docs/tutorials/basics.ipynb [hunk 1]
```diff
@@ -68,7 +68,7 @@
     "id": "67gUU740MSXd"
```
Comment (dabacon) [inline docs/tutorials/basics.ipynb:68]: <p>"These Device can be" fails grammar check.</p><p> </p><p>We are getting rid of devices in circuit next version, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (dabacon) [inline docs/tutorials/basics.ipynb:68]: <p>More uses of the word optimizer in this paragraph that should be replaced by transformers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (MichaelBroughton) [reply docs/tutorials/basics.ipynb:68]: <p>Yes, I've removed this language.</p>

Comment (MichaelBroughton) [reply docs/tutorials/basics.ipynb:68]: <p>Done.</p>

**Review by dabacon — APPROVED** [review]
LGTM

**Pull request merged**